framework/src/org/checkerframework/common/value/util/Range.java

```diff
@@ -130,6 +130,26 @@ public Range intRange() {
         }
     }
 
+    /**
+     * Either calls {@link #intRange()} if ignoreOverflow is false, or creates a new range whose
+     * from is either this.from or Integer.MIN_VALUE, and whose to is either this.to or
+     * Integer.MAX_VALUE.
+     */
+    public Range intRange(boolean ignoreOverflow) {
```

Comment:
Is there any reason that `ignoreOverflow` isn't a field of the Range class?  It seems error prone to have both this method and the `intRange()` method.  When should `intRange()` be called?  (The `intRange()` method should at least be private, but I think the better design would be for `ignoreOverflow` to be a final field.)